modules/manage/pages/manage-xdcr/secure-xdcr-replication.adoc

```diff
@@ -42,10 +42,23 @@ For step-by-step instructions, see xref:manage:manage-xdcr/enable-full-secure-re
 
 SCRAM-SHA is a multi-request protocol.
 The first request from the client (XDCR source to XDCR target) is responded to with a 401; the subsequent request completes the protocol.
-Therefore, when using half-secure replication, external monitoring or firewall software should allow (which is to say, ignore) these 401 responses; since they are part of the normal SCRAM-SHA protocol. 
+Therefore, when using half-secure replication, external monitoring or firewall software should allow (which is to say, ignore) these 401 responses; since they are part of the normal SCRAM-SHA protocol.
 
 If the monitoring or firewall software acts on these 401 responses by resetting or killing connections, SCRAM-SHA errors are displayed on the XDCR source cluster: if this occurs, check with your network monitoring or firewall administrators.
 
 If the monitoring or firewall software interferes with the XDCR connections, even though the XDCR replication will attempt to reconnect and continue to work through the connection interruptions, various issues may arise due to the continued interruptions -- including XDCR having to restart replications from sequence 0.
 
 Please note that because various XDCR processes make frequent calls to the target, in order to monitor for changes in topology and collection manifest, 401 responses associated with the SCRAM-SHA multi-request protocol will be seen by the external monitoring or firewall software continuously, while the replication is in progress.
+
+[#capella-trusted-cas]
+== Capella Trusted CAs
+
+CAs used by the Couchbase cloud data-platform, _Capella_, are automatically trusted, when _fully secure_ XDCR connections are made to Capella databases from Couchbase Enterprise Server 7.2+.
+This means that the Capella-database CA does _not_ need to be specified when references are configured to enable fully secure replication.
+Specifically, when a reference is being configured by means of:
+
+* Couchbase Web Console, the interactive pane, provided for specifying the CA, can be left blank.
+
+* The CLI, the `--xdcr-certificate` flag, provided for specifying the CA, does _not_ need to be used.
```

Comment:
Need to delete the line 62, since you still need to specify --xdcr-certifiate with couchbase-cli xdcr-setup until MB-56151 (couchbase-cli xdcr-setup does not allow trusting Capella certificates); it will not be fixed in 7.2

Reply:
Ah, okay. Will do.